packages/notte-agent/src/notte_agent/main.py

```
@@ -1,65 +1,47 @@
 from notte_core.credentials.base import BaseVault
-from notte_core.llms.engine import LlmModel
-from notte_sdk.types import DEFAULT_MAX_NB_STEPS, AgentCreateRequest
+from notte_sdk.types import AgentCreateRequestDict
 
 from notte_agent.common.base import BaseAgent
 from notte_agent.common.notifier import BaseNotifier, NotifierAgent
 from notte_agent.common.types import AgentResponse
-from notte_agent.falco.agent import FalcoAgent, FalcoAgentConfig
+from notte_agent.falco.agent import FalcoAgent
 from notte_agent.falco.types import StepAgentOutput
+from notte_agent.gufo.agent import GufoAgent
```

Ruff (0.11.9)

`notte_agent.gufo.agent.GufoAgent` imported but unused

Remove unused import: `notte_agent.gufo.agent.GufoAgent`

(F401)